### ISSUE TITLE
Availability on Windows?

### ISSUE THREAD

**Issue by jfix** [rimmartin/saxon-node#20]
Hello, I'm interested in using your bindings to Saxon/C, however I'm constrained to use Windows. Can you tell me what the current situation is? Can I use your module on Windows? Thanks for your help!

**Comment by ond1** [comment]
We will need a Saxon/C dll library build for windows. Which we are looking to do in the next release. Not sure what else is required.

**Comment by jfix** [comment]
Thanks for that quick answer. Any idea when that release might happen?
Thanks.

cheers,
Jakob.

**Comment by jfix** [comment]
Thanks for this (provisional) timeline!

I guess there is no help we can give to speed this up?  I'm really looking
for a way to use Saxon from node, and that seems to be the module we would
need, given that it's being maintained by you, the Saxonica people,
yourselves! ;-)

Again, thanks, and if we can provide help ...


cheers,
Jakob.

**Comment by ond1** [comment]
We could send you the dll library of the current release which you can then use with the C/C++ API o build the part that hooks up with saxon-node.

**Comment by jfix** [comment]
I'm afraid this is outside my competencies. :-( I'd be gladly testing the
module though! Thanks.

cheers,
Jakob.

**Comment by rimmartin** [comment]
I can test building saxon-node on a windows box

**Comment by ond1** [comment]
We have a Saxon/C 1.1.0 release out today which includes a windows release.

**Comment by rimmartin** [comment]
Ah great!  I'll update this interface and include windows!

**Comment by rimmartin** [comment]
Hi @ond1 

I needed to change enc_len to env_len on line 35 in SaxonCGlue.c to compile further.

Which Visual Studio are you using 2015?

With 2013 I run into the "SaxonProcessor.cpp(330): error C3861: 'snprintf': identifier not found" issue I see on the internet

We are close to saxon-node running again on Windows!

**Comment by nschonni** [comment]
CI was setup with (https://github.com/rimmartin/saxon-node/pull/22), but I haven't gotten it running on my own VS 2017 + Node 10 machine since there are hardcoded VS 2015 bits in the binding.gyp

**Comment by rimmartin** [comment]
you mean the version 
```
'msbuild_toolset': 'v120',
```
?

**Comment by nschonni** [comment]
Yes that's part of it, but removing that still didn't give me a successful build

**Comment by rimmartin** [comment]
can you post the first few compile error stacks?
